runtime/parachains/src/configuration.rs

```diff
@@ -215,6 +217,7 @@ impl<BlockNumber: Default + From<u32>> Default for HostConfiguration<BlockNumber
 			hrmp_max_parachain_outbound_channels: Default::default(),
 			hrmp_max_parathread_outbound_channels: Default::default(),
 			hrmp_max_message_num_per_candidate: Default::default(),
+			ump_max_individual_weight: 20 * WEIGHT_PER_MILLIS,
```

Comment:
I see what you mean however that won't make it better: all the other values are zero and this is basically a non-workable config. While it may be worth addressing, I think this should be out of scope for this PR